### PR TITLE
Instructions for fee calculation component

### PR DESCRIPTION
**Pull request by ksingh7**

### CODE REVIEW

File: pattern2-smart-city/deploy/README.adoc [hunk 1]
```diff
@@ -239,6 +239,22 @@ echo "http://$(oc get route | grep -i edge-kafdrop | awk '{print $2}')/topic/lpr
 echo "http://$(oc get route | grep -i core-kafdrop | awk '{print $2}')/topic/lpr/messages?partition=0&offset=0&count=100&keyFormat=DEFAULT&format=DEFAULT"
 ----
 
+=== Fee Calculation
+
+For calculating the toll and pollution fee, there are two cases that we have covered
```
Comment (guimou) [inline pattern2-smart-city/deploy/README.adoc:244]: Add : at the end

File: pattern2-smart-city/deploy/README.adoc [hunk 2]
```diff
@@ -239,6 +239,22 @@ echo "http://$(oc get route | grep -i edge-kafdrop | awk '{print $2}')/topic/lpr
...
+
+* When any vehichel enters the ULEZ, a certain fee (aka toll fee) must be applied to that vehicle
```
Comment (guimou) [inline pattern2-smart-city/deploy/README.adoc:246]: vehicle

**Review by guimou — COMMENTED** [review]
Two small typos, then can be merged.

File: pattern2-smart-city/deploy/README.adoc (hunk 1 above)
Comment (ksingh7) [reply pattern2-smart-city/deploy/README.adoc]: fixed

File: pattern2-smart-city/deploy/README.adoc (hunk 2 above)
Comment (ksingh7) [reply pattern2-smart-city/deploy/README.adoc]: fixed

**Pull request merged**